test.js

```diff
@@ -1,7 +1,72 @@
 import test from 'ava';
 import m from '.';
 
-test(t => {
-	t.is(m('5.1.2600'), 'XP');
-	t.is(m('10.0.10240'), '10');
+// Test using versions strings sourced from https://www.gaijin.at/en/lstwinver.php
```

Comment:
The runtime check to determine the version makes it impractical to have automated tests for this. And the way we can test this would mean that server versions would always return the ambiguous desktop versions:

Win 7, 8, 8.1 & 10 - https://msdn.microsoft.com/en-us/library/windows/desktop/ms724832(v=vs.85).aspx

Though if you had access to a server env that you could spin up a CLI in and test the module works as expected to boost confidence, that would be amazing :D